docs/docs/concepts/scalability_and_resilience.md

```diff
@@ -25,16 +25,16 @@ When a graceful shutdown request is received (SIGINT) an instance enters shutdow
 - gives any in-progress runs a limited number of seconds to finish (if not finished it will be put back in the queue)
 - stops the instance from picking up more runs from the queue
 
-If a hard shutdown occurs, eg. due to a server crash, or an infra failure, any runs that were in progress will be picked up by a periodic sweeper task that looks for in-progress runs that have breached their heartbeat window, which will put them back in the queue for another instance to pick them up.
+If a hard shutdown occurs due to a server crash or an infrastructure failure, any runs that were in progress will be picked up by a periodic sweeper task that looks for in-progress runs that have breached their heartbeat window, which will put them back in the queue for another instance to pick them up.
```

Comment:
```suggestion
If a hard shutdown occurs due to a server crash or an infrastructure failure, any runs that were in progress will be picked up by a periodic sweeper task that looks for in-progress runs that have breached their heartbeat window. The sweeper will put the runs back in the queue for another instance to pick them up.
```
Can we provide any more details here like how often the sweeper runs or if it's configurable / how you do it?

Reply:
Accepting your changes and adding a detail about the sweeper running every 2 minutes. It's not configurable. Do we need to explicitly say it's "not configurable"? I changed the adjective `periodic sweeper` to `internal sweeper` to try to convey that it's something that isn't exposed to the users and can't be touched/configured/etc.

Reply:
SGTM